### DISCUSSION TITLE
Pin the sidebar in closed position

### DISCUSSION THREAD

**Discussion by elle-ant** [outline/outline#7857] [Ideas]
I like to have multiple windows of Outline open at once and hide the sidebar to maximize content space. However, when mousing around, the sidebar often unhides and obscures the document, until I move my mouse to that window again. It would be nice if there were a way to keep it minimized in a non-auto-hide fashion.

**Comment by tommoor** [comment]
Looking at an alternative here where the sidebar will only open if the page has focus, I think that will do the trick without adding additional controls to manage? So if you just have a floating window on the desktop and are not actively focused within the menu will not swing open as the pointer moves over

**Reply by elle-ant** [reply]
What generally happens is when I switch working in an Outline window on the right half of my desktop to the left half of my desktop, the following occurs:

0. Right window is focused
1. Mouse moves from right half of the screen to the left, passing over the right window's sidebar
2. Right window's sidebar opens
3. Click in left window, focus shifts to left window
4. Right window's sidebar stays open.

To remedy this I have to:
1. Focus the right window
2. Move my mouse over the sidebar
3. Move my mouse over the main document
4. Return the mouse to the left side of the screen, avoiding revealing the sidebar.

Specifically, the following behaviors contribute to this issue:
* A window need not be focused for mouse hover to reveal the sidebar
* To hide the sidebar, the window must be focused, the mouse must be over/in the sidebar and then leave/move off it (it is not enough to just focus the window and have the mouse be off the sidebar.)

**Reply by elle-ant** [reply]
To that end I believe "Sidebar will switch to hidden (in auto-hide mode) if the window does not have focus" should address all these concerns
